_studio/mfx_lib/encode_hw/h265/src/mfx_h265_encode_vaapi.cpp

```diff
@@ -1294,6 +1294,8 @@ bool FillCUQPDataVA(Task const & task, MfxVideoParam &par, CUQPMap& cuqpMap)
 
     mfxU32 inputW = (par.m_ext.HEVCParam.PicWidthInLumaSamples   + inBlkSize - 1)/ inBlkSize;
     mfxU32 inputH = (par.m_ext.HEVCParam.PicHeightInLumaSamples  + inBlkSize - 1)/ inBlkSize;
+    inputW = mfx::align2_value(inputW, par.m_ext.HEVCParam.LCUSize / inBlkSize);
+    inputH = mfx::align2_value(inputH, par.m_ext.HEVCParam.LCUSize / inBlkSize);
 
```

Comment:
Check if (mbqp->NumQPAlloc < inputW * inputH) below can fail after change. Input buffer can be not aligned to LCU.